app-crm/app_crm/service.py

```
@@ -0,0 +1,156 @@
+        self.gateway.write_request_log(payload.request_id, log_payload)
+
+        return ImportListingsResponse(
+            request_id=payload.request_id,
+            processed_at=processed_at,
+            summary=summary,
+            duplicates=duplicates,
+        )
+
+    def _process_listing(
+        self,
+        idx: int,
```

Ruff (0.14.2)

Unused method argument: `idx`

(ARG002)